earth_enterprise/src/fusion/autoingest/tools/getop.cpp

```diff
@@ -245,9 +245,9 @@ main(int argc, char *argv[])
         outline("Number of cached assets: %u", taskLists.num_assets_cached);
         outline("Number of cached asset versions: %u",
                 taskLists.num_assetversions_cached);
-        outline("Total memory used by cached assets: %s",
+        outline("Approx. memory used by cached assets: %s",
```

Comment:
Very minor: it occurred to me that we could save a couple of lines in the getop output by putting the number of items and the size on the same line, like this: `Number of cached assets: <x>, approx. memory used: <y> B`. That way we could save 2 lines and print out more information about running and waiting tasks. Of course, this has nothing to do with this change, but I thought I'd mention it in case you're feeling ambitious.